.github/workflows/e2e-tests.yml

```diff
@@ -251,3 +251,11 @@ jobs:
     steps:
       - run: |
           echo "Didn't run due to conditional filtering"
+
+  pr-env:
+    needs: [build]
+    if: |
+      !cancelled() &&
+      contains(github.event.pull_request.labels.*.name, 'PR-Env')
+    uses: ./.github/workflows/pr-env.yml
+    secrets: inherit
```

Comment:
Why do we do this in E2E workflow?
Isn't https://github.com/metabase/metabase/blob/master/.github/workflows/uberjar.yml a better choice?

Reply:
So that we can use the uberjar artifacts that are built as part of the E2E workflow.

Reply:
Oh, I forgot that the uberjar.yml is building only on push to `master`.